src/libraries/System.Security.Cryptography.X509Certificates/tests/CertTests.cs

```diff
@@ -434,6 +434,41 @@ public static void SerializedCertDisposeDoesNotRemoveKeyFile()
             }
         }
 
+        [Fact]
+        public static void CopyResult_RawData()
+        {
+            using (X509Certificate2 cert = new X509Certificate2(TestData.MsCertificate))
+            {
+                byte[] first = cert.RawData;
+                byte[] second = cert.RawData;
+                Assert.NotSame(first, second);
+            }
+        }
+
+        [Fact]
+        public static void CopyResult_IssuerName()
+        {
+            using (X509Certificate2 cert = new X509Certificate2(TestData.MsCertificate))
+            {
+                X500DistinguishedName first = cert.IssuerName;
+                X500DistinguishedName second = cert.IssuerName;
+                Assert.NotSame(first, second);
```

Comment:
It looks like IssuerName (and probably SubjectName) does return the same reference on subsequent calls on .NET Framework; but mutation doesn't bleed through to the Issuer property.

The real question is "do people double-dip the property?" (which would create two copies of the RawData array).  If so, we probably want to just make the relationship between Issuer and IssuerName hardened, but just continue to have faith that callers don't alter the X500DistinguishedName.RawData value.  (I think I'd rather go with relationship-hardening, since it's equal footing with .NET Framework)

Reply:
I think I got the gist of what you mean. Pushed up some changes.